terragrunt/aws/glue/etl/bes/crm/salesforce.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You can delete this file - it'll get auto-generated once we have the ETL job defined.

Reply:
do I keep the folder path?

Reply:
Nope, you can delete the path as well - the ETL export workflow will create it for us.